internal/apischema/openai/openai.go

```diff
@@ -419,6 +428,8 @@ type ChatCompletionAssistantMessageParamContent struct {
 	Refusal *string `json:"refusal,omitempty"`
 	// The text content.
 	Text *string `json:"text,omitempty"`
+
+	RedactedContent []byte `json:"redactedContent,omitempty"`
```

Comment:
I am understanding correctly that this is the AWS-only vender specific param right? shouldn't we need the same structure as the `*GCPVertexAIVendorFields` inlined at the top level here?

Or should we somehow define it in the toplevel `AWSVendorFields` on ChatCompletionRequest struct? In other words, I think we haven't decided on what to do with non-top level vendor specific fields? cc @johnugeorge @sukumargaonkar @yuzisun